src/WebJobs.Script/ExtensionBundle/IExtensionBundleManager.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.ExtensionBundle
+{
+    public interface IExtensionBundleManager
+    {
+        Task<string> GetExtensionBundle(HttpClient httpClient = null);
```

Comment:
Based on the comment above, just have two overloads of this method, with an without the client argument